tracer/src/Datadog.Trace/AppSec/Waf/Initialization/LibraryLoader.cs

```diff
@@ -182,12 +182,12 @@ private static bool TryLoadLibraryFromPaths(string libName, List<string> paths,
             return success;
         }
 
-        private static string GetLibName(FrameworkDescription fwk)
+        private static string GetLibName(FrameworkDescription fwk, string libVersion = null)
             => fwk.OSPlatform switch
             {
-                OSPlatformName.MacOS => "libddwaf.dylib",
-                OSPlatformName.Linux => "libddwaf.so",
-                OSPlatformName.Windows => "ddwaf.dll",
+                OSPlatformName.MacOS => $"libddwaf{libVersion}.dylib",
+                OSPlatformName.Linux => $"libddwaf{libVersion}.so",
+                OSPlatformName.Windows => $"ddwaf{libVersion}.dll",
```

Comment:
Maybe add an hyphen to separate the version if specified?

Reply:
fixed in https://github.com/DataDog/dd-trace-dotnet/pull/3251/commits/596ff9793552dbd57e2effa878427e1d4e4db209, it was all to avoid the if 😅